nms/app/packages/magmalte/app/state/AlarmsApiUtil.js

```diff
@@ -39,11 +39,30 @@ export const MagmaAlarmsApiUtil: ApiUtil = {
     console.warn('not implemented');
     return [];
   },
-  getTroubleshootingLink: async ({alertName: _}) =>
-    Promise.resolve({
-      link: '',
+  getTroubleshootingLink: async ({alertName}) => {
+    const alertLinks = [
+      'S1 Setup Failure',
+      'UE attach Failure',
+      'Gateway Checkin Failure',
+      'eNodeB Failure To Connect',
+      'No Metrics Available',
+      'S6A Authentication Success Rate',
+      'Service Request Success Rate',
+      'AGW Reboot',
+      'Expected Service Restart',
+      'Unexpected Service Restart',
+    ];
+    const alert = alertName.toLowerCase();
+    const link = alertLinks.includes(alertName)
```

Comment:
If the alert links doesn't contain the alert name, should we redirect them to general set of troubleshooting guide instead of empty string here

Reply:
Right now I'm not showing any link if it is an empty string

Reply:
Let's defer this decision to @jingtien